airbyte-integrations/connectors/source-sendgrid/source_sendgrid/sendgrid.yaml

```diff
@@ -0,0 +1,331 @@
+page_size: 50
+
+schema_loader:
```

Comment:
an advice here is that you should have a top level `definitions` where you can create your reusable objects. Right now it is polluting the top level "namespace" and that will make it harder to extend it.

Reply:
that could be a part of the spec.

Reply:
yes! done